### PR TITLE
Docker nginx config: use mime.types and extend by image/avif

### PR DESCRIPTION
**Pull request by brusch**

### CODE REVIEW

**Comment by brusch** [comment]
See also https://github.com/pimcore/demo/pull/314

File: .docker/nginx.conf [hunk 1]
```diff
@@ -1,7 +1,8 @@
-# mime types are covered in nginx.conf by:
-# http {
-#   include       mime.types;
-# }
+
+include mime.types;
```
Comment (jdreesen) [inline .docker/nginx.conf]: Is this really necessary? It should already be present through the (default) `nginx.conf` (as the comment previously said).

File: .docker/nginx.conf [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+types {
+    image/avif avif;
```
Comment (jdreesen) [inline .docker/nginx.conf:5]: Note, that this was [already fixed upstream](https://github.com/nginx/nginx/blob/master/conf/mime.types#L18), it's just not in the `stable` release, yet (it's in the `mainline` one, though).

Comment (brusch) [reply .docker/nginx.conf:5]: Yep, have seen that ... but actually it doesn't help for the moment 😉

File: .docker/nginx.conf (hunk 1 above)
Comment (brusch) [reply .docker/nginx.conf]: You're right, removed it. Thanks!

File: .docker/nginx.conf (hunk 2 above)
Comment (jdreesen) [reply .docker/nginx.conf:5]: True, unless you want to change the image from `nginx:stable-alpine` to `nginx:alpine` ;)

**Pull request merged**